### PR TITLE
1.0.3: bugfix, deleted projects appearing on dashboard

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

Companion PR to https://github.com/cityofaustin/atd-moped/pull/582

Fixes bug and updates the version in package.json

## Testing
**URL to test:** 

**Steps to test:**

---
#### Ship list
- [ ] Code reviewed 
- [x] Product manager approved
- [x] ~Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)~

### CODE REVIEW

**Review by amenity — APPROVED** [review]

File: moped-editor/src/queries/signals.js [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
 export const SIGNAL_PROJECTS_QUERY = gql`
   query SignalProjectsQuery {
     moped_project(
       where: {
+        is_retired: { _neq: true }
         _or: [
           {
             moped_proj_components: {
               moped_components: { component_name: { _ilike: "signal" } }
+              status_id: {_eq: 1}
```
Comment (frankhereford) [inline moped-editor/src/queries/signals.js:14]: This is totally fine, but like we were discussing, we may want to consider a look-up table to get these magic numbers out.

**Review by frankhereford — APPROVED** [review]
🚢 🚢 

Thank you Chia!

**Comment by johnclary** [comment]
Thank you Chia!

File: moped-editor/src/queries/signals.js (hunk 1 above)
Comment (mateoclarke) [reply moped-editor/src/queries/signals.js:14]: agreed. Should we go ahead and make a tech debt issue for this? cc/ @amenity @johnclary

**Review by mateoclarke — APPROVED** [review]
👍

**Comment by amenity** [comment]
@mateoclarke @frankhereford - would/could that be addressed here? https://github.com/cityofaustin/atd-data-tech/issues/8238

**Comment by mateoclarke** [comment]
> @mateoclarke @frankhereford - would/could that be addressed here? [cityofaustin/atd-data-tech#8238](https://github.com/cityofaustin/atd-data-tech/issues/8238)

yeah I think so!

**Pull request merged**